src/main/scala/net/liftmodules/ng/Angular.scala

```diff
@@ -62,15 +62,26 @@ object Angular extends DispatchSnippet with AngularProperties with LiftNgJsHelpe
       case "Angular" => this
       case "i18n" => AngularI18n
     }
-    
+
     LiftRules.addToPackages("net.liftmodules.ng")
 
     ResourceServer.allow {
       case "net" :: "liftmodules" :: "ng" :: "js" :: _ => true
     }
 
+    LiftRules.responseTransformers.append { res =>
+      import net.liftweb.util.Helpers._
+
+      res match {
+        case r: XhtmlResponse =>
+          val body = (s"$appSelector [expose-scope]" #> "").apply(r.out).headOption.getOrElse(r.out)
+          r.copy(out = body)
+        case _ => res
+      }
+    }
+
```

Comment:
Not very fond of doing this. I'm thinking it'll be best to add a README note for the developer to add `expose-scope` manually. I figure folks trying to disable debug mode will be the same folks who will find such a note. Also, I don't like the idea of Lift doing more work to support production mode, when it can be easily added to the template and forgotten about. If we got this route, I might create a directive named `lift-ng` in case we need to hook other things in the future.